### PR TITLE
Fix/#1339 consolidate spinners

### PR DESCRIPTION
**Pull request by Mi-Lan**

### CODE REVIEW

File: app/src/components/common/icons/IconSpinner2.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+interface Props {
+  spinnerSize?: string
+}
+
+export const IconSpinner = (props: Props) => {
```
Comment (hexyls) [inline app/src/components/common/icons/IconSpinner2.tsx:7]: Nice work!

File: app/src/components/common/spinner/index.tsx [hunk 2]
```diff
@@ -1,8 +1,7 @@
 import React, { HTMLAttributes } from 'react'
 import styled, { keyframes } from 'styled-components'
 
-import SpinnerSVG from './img/spinnerMain.svg'
```
Comment (hexyls) [inline app/src/components/common/spinner/index.tsx:4]: If we aren't using the `spinnerMain.svg` or `spinnerMainBig.svg` files anymore we can probably delete them. Thoughts?

File: app/src/components/common/spinner/index.tsx [hunk 3]
```diff
@@ -13,37 +12,24 @@ const rotate = keyframes`
...
   color?: string
-  height?: string | undefined
-  width?: string | undefined
+  size?: string
   big?: boolean | undefined
 }
 
 export const Spinner: React.FC<Props> = (props: Props) => {
-  const { big, color = '#fff', height, width, ...restProps } = props
+  const { big, color = '#fff', size, ...restProps } = props
```
Comment (hexyls) [inline app/src/components/common/spinner/index.tsx:28]: Nice!

File: app/src/components/common/icons/index.ts [hunk 4]
```diff
@@ -23,3 +23,4 @@ export { IconSearchGrey } from './IconSearchGrey'
 export { IconKleros } from './IconKleros'
 export { IconExclamation } from './IconExclamation'
 export { IconSchedule } from './IconSchedule'
+export { IconSpinner } from './IconSpinner2'
```
Comment (hexyls) [inline app/src/components/common/icons/index.ts:26]: What do you think about renaming this file from `IconSpinner2` to just `IconSpinner`?

**Review by hexyls — APPROVED** [review]

**Review by liaojikunwork — APPROVED** [review]

File: app/src/components/common/spinner/index.tsx (hunk 2 above)
Comment (Mi-Lan) [reply app/src/components/common/spinner/index.tsx:4]: yeah definitely good catch!

File: app/src/components/common/icons/index.ts (hunk 4 above)
Comment (Mi-Lan) [reply app/src/components/common/icons/index.ts:26]: well that's where i had problem i don't know why but when i rename it to just IconSpinner it converts to like text file and it's not usable...

**Review by pimato — APPROVED** [review]

File: app/src/components/common/icons/index.ts (hunk 4 above)
Comment (Mi-Lan) [reply app/src/components/common/icons/index.ts]: @hexyls 
Screenshot 2020-11-24 at 12 27 49
here is my concern when i change the name you see how IconSpinner next to it it's like text file and also this error like it's not exported good...

**Review by liaojikunwork — APPROVED** [review]

**Review by KadenZipfel — APPROVED** [review]

File: app/src/components/common/icons/index.ts (hunk 4 above)
Comment (KadenZipfel) [reply app/src/components/common/icons/index.ts]: Is this just an issue with your IDE identifying the file type? Seems to work fine for me

**Pull request merged**